go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/dynatrace-oss/opentelemetry-metric-go
 go 1.15
 
 require (
+	github.com/dynatrace-oss/dynatrace-metric-utils-go v0.0.0-20210504070650-4848b0a500cb
 	go.opentelemetry.io/otel v0.13.0
```

Comment:
Updating to latests otel (0.20.0) seems not as straightforward as I'd hoped. Some things have been moved around and the metrics impl seems to be completely different.